### ISSUE TITLE
`Remove all builtin apps` will remove WSL

### ISSUE THREAD

**Issue by gzz2000** [builtbybel/privatezilla#34]
.. which got me in trouble...... Now I have lost some files within WSL1 filesystem, and all of the environments I configured in it and I have to reinstall it completely
Please consider adding WSL to some whitelist

**Comment by Belim** [comment]
WSL is not distributed via APPX file format which is used for installung/uninstallung UWP/Modern apps.

Snce Privatezilla only removes appx packages Ican't imagine that at all. Maybe some dependencies have been removed. But i wouldn't know which ones

**Comment by gzz2000** [comment]
> So WSL is distributed via APPX file format which is used for installing/uninstalling UWP/Modern apps!? Didnt know that ... Hmm
> 
> Did you install your Linux Distro via the Microsoft store?

Yes. I installed WSL1 ubuntu 16.04 via MS store. The filesystem lies in `%APPDATA%\Local\Packages\CanonicalGroupLimited.UbuntuonWindows_79rhkp1fndgsc\LocalState\rootfs`.
This directory is deleted completely after the removal of apps.

**Comment by lhoe** [comment]
Same Issue here. Everything got deleted.       
Afterwards I reinstalled WSL manually ( see https://docs.microsoft.com/de-de/windows/wsl/install-manual ) and ran the tool again. Again everything was deleted.   

Ideally there would be an output that shows which of the detected apps will get deleted.

**Comment by MrDetonia** [comment]
Just ran into this myself and lost a bunch of ssh keys. 🤦  
+1 to a summary of which apps will be deleted, or some sort of sensible blacklist that can be updated with known bloatware store apps.

**Comment by Bondrake** [comment]
Same thing just happened here. Not fun. Please fix.

**Comment by brucosper** [comment]
Same issue here. At the very least a warning would be nice.
